identity/handler.go

```diff
@@ -219,19 +248,67 @@ type AdminCreateIdentityBody struct {
 	// required: true
 	Traits json.RawMessage `json:"traits"`
 
+	// Password for the user. This will only be used if the json schema actually defines the
+	// password as a valid type.
+	//
+	// required: false
+	Password *PasswordCredential `json:"password"`
+
 	// State is the identity's state.
 	//
 	// required: false
 	State State `json:"state"`
+
+	// VerifiableAddresses contains all the addresses that can be verified by the user.
+	//
+	// required: false
+	VerifiableAddresses []VerifiableAddress `json:"verifiable_addresses"`
```

Comment:
The problem here will be that if the email from the traits must match the email set here, otherwise the entry will be deleted immediately. I think we need at least some type of validation here.

What should we do with the extra fields here? So `Via`, `Status`, ...?